code/workspaces/client-api/package.json

```diff
@@ -37,13 +37,15 @@
     "eslint-config-airbnb-base": "^13.2.0",
     "eslint-plugin-import": "^2.18.2",
     "jest": "^24.8.0",
-    "regenerator-runtime": "^0.13.3"
+    "nodemon": "^1.19.1",
+    "regenerator-runtime": "^0.13.3",
+    "shx": "^0.3.2"
   },
   "scripts": {
     "test": "jest --watch",
     "test-ci": "CI=true jest",
     "lint": "eslint ./src",
-    "build": "rm -rf dist && babel src --root-mode upward-optional --ignore **/__mocks__/,**/*.spec.js --out-dir dist",
-    "start": "babel-node --root-mode upward --ignore='../../node_modules' ./src/app.js"
+    "build": "shx rm -rf dist && babel src --root-mode upward-optional --ignore **/__mocks__/,**/*.spec.js --out-dir dist",
+    "start": "nodemon --watch src --ext js --ignore **/*.spec.js --exec babel-node --root-mode upward --ignore='../../node_modules' ./src/app.js"
```

Comment:
Do you need the `--watch src --ext js`? Aren't these the defaults?

Reply:
Yes, you are right. Will fix these.